dao/src/main/java/greencity/entity/order/TariffsInfo.java

```diff
@@ -52,17 +52,11 @@ public class TariffsInfo {
     @Column(nullable = false)
     private LocalDate createdAt;
 
-    @Column(name = "min_amount_of_big_bags")
-    private Long minAmountOfBigBags;
```

Comment:
We have got `LimitsDto` class with the same fields but it never used
Please remove it